### ISSUE TITLE
Comment added by `add` plugin not placed in the top of file

### ISSUE THREAD

**Issue by vhenzl** [dotansimha/graphql-code-generator#1890]
In the recent canary version (`1.1.4-alpha-62ee81ed.50`), comments added by `add` plugin are after other content, even if the plugin is listed as the first one in the config.

Particularly, `Maybe` type and imports (when `near-operation-file-preset` used) precede the `add` output:

```ts
export type Maybe<T> = T | null;
// DO NOT EDIT
/** All built-in and custom scalars, mapped to their actual values */
export type Scalars = {
  ID: string;
  String: string;
  Boolean: boolean;
  Int: number;
  Float: number;
};
```

**`codegen.yml` config file:**

```yml
overwrite: true
schema: "schema.graphql"
generates:
  src/generated/graphql.ts:
    plugins:
      - "add": "// DO NOT EDIT"
      - "typescript"
```

**Expected behaviour:**

The output of `add` plugin is in the top of the generated file.

**Environment:**

- OS: Win 10
- NodeJS: v10.15.3

**Comment by dotansimha** [comment]
Working on a fix now :)

**Comment by dotansimha** [comment]
Fixed in: https://github.com/dotansimha/graphql-code-generator/commit/28b96fc13c6a52de47f6e33c45a87cbe501349aa 
@vhenzl , can you please `1.1.4-alpha-6b4dc6f2.54`?

**Comment by vhenzl** [comment]
Yes, it works! 👍 Thx.

**Comment by dotansimha** [comment]
The fix is available in 1.2.0.
